src/OndaEDF.jl

```diff
@@ -1,26 +1,129 @@
 module OndaEDF
 
 using Base.Iterators
-using Compat: @compat
-using Dates, UUIDs
-using Onda, EDF
+using Dates
+using EDF
+using Legolas
+using Onda
+using PrettyTables
 using StatsBase
 using TimeSpans
 using Tables
+using UUIDs
+
+using Legolas: @row, lift
+using Onda: LPCM_SAMPLE_TYPE_UNION, onda_sample_type_from_julia_type, convert_number_to_lpcm_sample_type
 using Tables: rowmerge
 
+export write_plan
+export edf_to_onda_samples, edf_to_onda_annotations, plan_edf_to_onda_samples, store_edf_as_onda
+export onda_to_edf
+
 # can be dropped if we drop Onda<0.14
 sample_type(x) = isdefined(Onda, :sample_type) ? Onda.sample_type(x) : x.sample_type
 
 include("standards.jl")
 
+"""
+    Plan = @row("ondaedf.plan@1",
+                # EDF.SignalHeader fields
+                label::String
+                transducer_type::String
+                physical_dimension::String
+                physical_minimum::Float32
+                physical_maximum::Float32
+                digital_minimum::Float32
+                digital_maximum::Float32
+                prefilter::String
+                samples_per_record::Int16
+                # EDF.FileHeader field,
+                seconds_per_record::Float64
+                # Onda.SamplesInfo fields (channels -> channel), may be missing
+                kind::Union{Missing, AbstractString}
+                channel::Union{Missing, AbstractString}
+                sample_unit::Union{Missing, AbstractString}
+                sample_resolution_in_unit::Union{Missing, LPCM_SAMPLE_TYPE_UNION}
+                sample_offset_in_unit::Union{Missing, LPCM_SAMPLE_TYPE_UNION}
+                sample_type::Union{Missing, AbstractString}
+                sample_rate::Union{Missing, LPCM_SAMPLE_TYPE_UNION}
+                # errors, use `nothing` to indicate no error
+                error::Union{Nothing, String})
```

Comment:
The documentation of these schemas might be useful to have in the `README.md` to make it easier to find?

Reply:
The README covers their contents in way, but I would still add a note about their name (both `Plan` and `ondaedf.plan`) for breadcrumbs on where to look for the full details. :smile:

Reply:
added an explicit call out for the schemas and to `write_plan`.